worker/backend/backend.go

```diff
@@ -20,6 +20,12 @@ type Backend struct {
 	namespace string
 }
 
+type InputValidationError struct{}
```

Comment:
I wonder if we should get these down to an `errors.go` file 🤔  seems a bit disconnected from the purpose `backend.go`

wdyt?

Reply:
runtime package has a types.go file in the `new-put-get-step` branch. I'd move it if there are many more errors we plan on adding.

Reply:
oh true - I forgot about the new `runtime` package. That would be a nice place to put error types, as a source of truth for all common abstractions in runtime world 😎  We can do this after the refactor is merged in.